### PR TITLE
[SYNC] [WW16] [skip-lab] student feedback, CI duplication

### PR DESCRIPTION
**Pull request by demid5111**

### CODE REVIEW

File: config/_stage_run_pytest.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+if [[ "$LAB_PATH" == "lab_5_scrapper" ]]; then
+  python config/config_param_changer.py --config_path="lab_5_scrapper/scrapper_config.json"
+  echo "Changed config params"
+fi
+
+if [[ "$LAB_PATH" == "lab_5_scrapper" ]]; then
+  python -m pytest -m "mark${TARGET_SCORE} and ${PYTEST_LABEL}" --capture=no --ignore=lab_6_pipeline
+else
+  if [[ ${TARGET_SCORE} != 10 ]]; then
+    python -m pytest -m "mark${TARGET_SCORE} and ${PYTEST_LABEL}" --capture=no --ignore lab_6_pipeline/tests/s4_pos_frequency_pipeline_test.py --ignore lab_6_pipeline/tests/s3_6_advanced_pipeline_test.py
```
Comment (demid5111) [inline config/_stage_run_pytest.sh:28]: why do we need ignoring of particular files?

File: config/_stage_run_pytest.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+ret=$?
+if [ "$ret" = 5 ]; then
+  echo "No tests collected.  Exiting with 0 (instead of 5)."
+  exit 0
+fi
+
+echo "Pytest results (should be 0): $ret"
+
+exit "$ret"
```
Comment (demid5111) [inline config/_stage_run_pytest.sh:42]: no newline

File: config/stage_2_crawler_tests/_stage_collect_articles.sh [hunk 3]
```diff
@@ -14,27 +16,19 @@ echo "Changed config params"
...
-echo "Checking volume of files"
-
-TARGET_SCORE=$(bash config/get_mark.sh lab_5_scrapper)
-
-if [[ ${TARGET_SCORE} == 4 ]]; then
-  echo "Running score four checks"
-  python -m pytest -m "mark4 and stage_2_4_dataset_volume_check" --capture=no --ignore=lab_6_pipeline
+if [[ $? -ne 0 ]]; then
+  echo "Check failed for common files."
+  FAILED=1
```
Comment (demid5111) [inline config/stage_2_crawler_tests/_stage_collect_articles.sh:21]: exit immediately

File: core_utils/tests/utils.py [hunk 4]
```diff
@@ -61,5 +61,5 @@ class TestInputs:
                     '13\tлет\tгод\tNOUN\t_\t_\t0\troot\t_\t_',
                     '14\tс\tс\tADP\t_\t_\t0\troot\t_\t_',
                     '15\tпочтой\tпочта\tNOUN\t_\t_\t0\troot\t_\t_',
-                    '16\ttest\ttest\tNOUN\t_\t_\t0\troot\t_\t_',
+                    '16\ttest\ttest\tX\t_\t_\t0\troot\t_\t_',
```
Comment (demid5111) [inline core_utils/tests/utils.py:64]: should be a reference file and not giant strings (!)

File: config/unpack_archived_dataset.sh [hunk 5]
```diff
@@ -3,19 +3,22 @@ set -ex
...
 
-if [[ ${TARGET_SCORE} != 0 ]]; then
+if [[ ${TARGET_SCORE_SCRAPPER} != 0 || ${TARGET_SCORE_PIPELINE} != 0 ]]; then
   mkdir -p tmp/articles
-  mv *_cleaned.txt tmp/articles
   mv *_raw.txt tmp/articles
+  mv *_cleaned.txt tmp/articles || echo "no files to move"
 
-  if [[ ${TARGET_SCORE} != 4 ]]; then
+  if [[ ${TARGET_SCORE_SCRAPPER} -gt 4 || ${TARGET_SCORE_PIPELINE} -gt 4 ]]; then
```
Comment (demid5111) [inline config/unpack_archived_dataset.sh:15]: I do not agree - why do we start caring about lab 5 in this script?

File: docs/public/ud_format.md [hunk 6]
```diff
@@ -113,6 +113,9 @@ the resulting string for our example would be:
 > **NB**: The complete mapping of features from Mystem to the UD format can be found in
 > the [`ud_mapping.md`](ud_mapping.md).
 
+> **HINT**: It's mandatory to use `Loc` mapping for pymystem3 `пр` case because
+> in Slavic languages this is the only case that is used exclusively in combination with prepositions.
```
Comment (demid5111) [inline docs/public/ud_format.md:117]: you said there is an explanation in the official docs, it would be more persuasive to put it here

**Review by demid5111 — COMMENTED** [review]
to discuss on the sync (@NovIrina)

File: config/unpack_archived_dataset.sh (hunk 5 above)
Comment (NovIrina) [reply config/unpack_archived_dataset.sh:15]: was fixed by Vita

File: docs/public/ud_format.md (hunk 6 above)
Comment (NovIrina) [reply docs/public/ud_format.md:117]: fixed

File: config/stage_2_crawler_tests/_stage_collect_articles.sh (hunk 3 above)
Comment (NovIrina) [reply config/stage_2_crawler_tests/_stage_collect_articles.sh:21]: fixed

File: config/_stage_run_pytest.sh (hunk 2 above)
Comment (NovIrina) [reply config/_stage_run_pytest.sh:42]: fixed

File: config/_stage_run_pytest.sh (hunk 1 above)
Comment (NovIrina) [reply config/_stage_run_pytest.sh:28]: fixed

**Pull request merged**